python/extractor/tests/parser/.gitignore

```diff
@@ -0,0 +1 @@
+*.old
```

Comment:
Maybe you'll want `*.new` as well here? Both those file types are generated during testing, after all.

Reply:
Yes, I did consider that, let's add it.